airbyte-integrations/connectors/source-braze/source_braze/transformations.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass
+from typing import Optional
+
+import dpath
+from airbyte_cdk.sources.declarative.transformations import AddFields
+from airbyte_cdk.sources.declarative.types import Record, Config, StreamState, StreamSlice
+
+
+@dataclass
+class TransformToRecordComponent(AddFields):
```

Comment:
please create a Github issue for this custom component as well so we can keep track of which components we need to add to the CDK